### ISSUE TITLE
Trying to forward message

### ISSUE THREAD

**Issue by miaow2** [LonamiWebs/Telethon#101]
Hello,

I try to forward last message from bot (entities[1]) to supergroup (entities[2]), but I get an error:

```
[01:14:47.902/MainThread] WARNING: Read RPC error: (RPCError(...), 'Unknown error message with code 400: PEER_ID_INVALID')
Unexpected error (<class 'telethon.errors.RPCError'>): (RPCError(...), 'Unknown error message with code 400: PEER_ID_INVALID') at
Traceback (most recent call last):
  File "/root/Telethon/try_telethon_test.py", line 38, in <module>
    client.run()
  File "/root/Telethon/telethon/interactive_telegram_client_test.py", line 103, in run
    self.invoke(ForwardMessageRequest(get_input_peer(entities[2]), msg.id, generate_random_long()))
  File "/root/Telethon/telethon/telegram_client.py", line 170, in invoke
    self.sender.receive(request, timeout)
  File "/root/Telethon/telethon/network/mtproto_sender.py", line 151, in receive
    request)
  File "/root/Telethon/telethon/network/mtproto_sender.py", line 229, in process_msg
    return self.handle_rpc_result(msg_id, sequence, reader, request)
  File "/root/Telethon/telethon/network/mtproto_sender.py", line 362, in handle_rpc_result
    raise error
telethon.errors.RPCError: (RPCError(...), 'Unknown error message with code 400: PEER_ID_INVALID')
```

There is a code:

        dialog_count = 4
        dialogs, entities = self.get_dialogs(dialog_count)
        total_count, messages, senders = self.get_message_history(entities[1], limit=1)
        for msg, sender in zip(reversed(messages), reversed(senders)):
            name = sender.first_name if sender else '???'
            if getattr(msg, 'media', None):
                self.found_media.add(msg)
                content = '<{}> {}'.format(msg1.media.__class__.__name__, getattr(msg1.media, 'caption', ''))
            elif hasattr(msg. 'message'):
                content = msg.message
            elif hasattr(msg, 'action'):
                content = str(msg.action)
            else:
                content = msg.__class__.__name__
            self.invoke(ForwardMessageRequest(get_input_peer(entities[2]), msg.id, generate_random_long()))

**Comment by Lonami** [comment]
Okay Telegram is a bit weird with channels. Stop using `ForwardMessageRequest` (note it's *Message*) and use `ForwardMessagesRequest` (note it's _Message**s**_). So it would be something like, for a single `msg`:

```python
client.invoke(ForwardMessagesRequest(
    from_peer=get_input_peer(entities[1]),
    id=[msg.id],
    random_id=[generate_random_long()],
    to_peer=get_input_peer(entities[2])
))
```

I added the named arguments for clarity although they're not needed because I wrote them in order. Check out [`ForwardMessagesRequest`](https://lonamiwebs.github.io/Telethon/methods/messages/forward_messages.html) for the full list of available parameters.
